runner.sh

```diff
@@ -1,30 +1,43 @@
 #!/bin/bash
+
 set -e
 
-sudo touch requirements.txt
 user=nsls2
-image="condaforge"
+image="nsls2-analysis-2021-1.2"
 timestamp=$(date +%Y%m%d%H%M%S)
 env_name="nsls2-analysis-2021-1.2"
-#docker build . -t conda --target build
-#docker build . -t temp-image --target export
-#docker image build . -t $user/${image}:latest \
- #                  -t $user/${image}:${timestamp}
-#docker run -it --rm -v $PWD:/build $user/${image}:${timestamp} bash -l /build/export.sh
-#docker create -ti --name temp-container temp-image bash
-#docker cp temp-container:/build .
+
+if [ -f "nsls2-analysis-2021-1.2.tar.gz" ]
+then
+        read -p "These tar.gz and yml files already exist. Would you like to replace the existing files? " REPLACE
+        if [ "$REPLACE" == 'yes' ] || [ "$REPLACE" == 'y' ]
```

Comment:
```suggestion
        if [ "$REPLACE" == 'yes' -o "$REPLACE" == 'y' ]
```

Reply:
I took these parts out for now and just went with your original suggestion. I will probably add them back in tomorrow.

Leaving this unresolved for now.

Reply:
I took out all interactive inputs.